packages/launchpad/src/container/style.scss

```diff
@@ -0,0 +1,57 @@
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/base-styles/breakpoints";
+@import "@automattic/typography/styles/variables";
+
+.launchpad-container {
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+}
+
+// Launchpad Content.
+.launchpad-container__content {
+	display: flex;
+	flex-direction: column;
+
+	@include break-large {
+		height: 100%;
+		flex-direction: row;
+	}
+}
+
+// Launchpad Sidebar.
+.launchpad-container__sidebar {
+	max-width: 100%;
+	margin: 0 16px 32px;
+	display: flex;
+	flex-direction: column;
+
+	@include break-large {
+		max-width: 360px;
+		margin: 0 20px 24px 24px;
+		gap: 80px;
+	}
+
+	// TODO: Migrate to the checklist components. The checklist could receive a special className as prop that would be responsible for this.
```

Comment:
For now, I kept it as it is, but it would be nice to change it at some point to be a responsibility of the checklist.